### PR TITLE
feat : Product page

### PR DESCRIPTION
**Pull request by Tanjum786**
Listed products on the product page 

1. used mockbee API 
2. Listed products and the category products

### CODE REVIEW

**Review by RahulSingh9131 — APPROVED** [review]
checked your code, everything is looking good to me. code structure is clean and understandable, contains nice use of pre-defined variables in css.

File: src/pages/Cart/Cart.jsx [hunk 1]
```diff
@@ -15,12 +15,12 @@ export const Cart = () => {
...
-          DeliveryAmt={"₹0"}
-          SaveAmt={"₹250"}
-          TotalAmt={"₹250"}
-          cartitem={"1"}
+          price={250}
+          DiscAmt={250}
+          DeliveryAmt={0}
+          SaveAmt={250}
+          TotalAmt={250}
+          cartitem={1}
```
Comment (shuntzhere) [inline src/pages/Cart/Cart.jsx:23]: You might not need to put these inside the curly braces. DO check once

File: src/pages/Landing-page/Component/Productcategory.jsx [hunk 2]
```diff
@@ -1,15 +1,27 @@
+import axios from "axios";
 import { Link } from "react-router-dom";
+import React, { useState, useEffect } from "react";
 
-const Productcategory = ({ cardimg, cardAlt, cardTitle }) => {
+const Productcategory = () => {
+  const [dataCat, setdataCata] = useState([]);
```
Comment (shuntzhere) [inline src/pages/Landing-page/Component/Productcategory.jsx]: This should be [dataCata]

File: src/pages/ProductLists/Productlisting.jsx [hunk 3]
```diff
@@ -1,50 +1,18 @@
...
 import "./productpage.css";
+
 export const Productlisting = () => {
   return (
     <>
       <div>
-      <Navbar auth={"Login"} linkto={'/login'}/>
+        <Navbar auth={"Login"} linkto={"/login"} />
         <section className="product_container dis_flex">
           <Filtters />
```
Comment (shuntzhere) [inline src/pages/ProductLists/Productlisting.jsx]: This might be a typo, do check.

**Review by shuntzhere — APPROVED** [review]
Suggested a few changes, Try to give image links on your database itself. No need to import it from other files.

File: src/pages/Cart/Cart.jsx (hunk 1 above)
Comment (Tanjum786) [reply src/pages/Cart/Cart.jsx:23]: ok i will fix it

**Pull request merged**